### ISSUE TITLE
Sending notification workspace/didChangeConfiguration failed

### ISSUE THREAD

**Issue by sschuh** [microsoft/AL#7664]
**1. Describe the bug**
Everytime i open a workspace i get the following error message.

<img width="343" alt="image" src="https://github.com/microsoft/AL/assets/34685182/55b7120e-4104-4da5-9b60-a14c884a7979">

[Error - 9:41:18 AM] Sending notification workspace/didChangeConfiguration failed

**2. To Reproduce**
Open a workspace file.

```
{
	"folders": [
		{
			"name": ".alpackages",
			"path": ".alpackages"			
		},
		{
			"name": "App",
			"path": "app"			
		},
		{
			"name": "TestApp",
			"path": "testapp"
		}
	],
	"settings": {
	}
}
```

**3. Expected behavior**
No error message :(

**4. Actual behavior**
Error message.

**5. Versions:**

- AL Language: 
v13.0.937154 (pre-release)

- Visual Studio Code:
Version: 1.86.0 (user setup)
Commit: 05047486b6df5eb8d44b2ecd70ea3bdf775fd937
Date: 2024-01-31T10:28:19.990Z
Electron: 27.2.3
ElectronBuildId: 26495564
Chromium: 118.0.5993.159
Node.js: 18.17.1
V8: 11.8.172.18-electron.0
OS: Windows_NT x64 10.0.22621

- Business Central:
BC23.4

- List of Visual Studio Code extensions that you have installed:
<img width="377" alt="image" src="https://github.com/microsoft/AL/assets/34685182/67ac2523-93fa-491e-8722-1289d74b261b">

**Comment by javisoft** [comment]
I had the same issue, and it was a typo within the app.json file. I found it by looking at the git pending changes. The app.json was like this: 

<img width="252" alt="image" src="https://github.com/microsoft/AL/assets/346940/b936d6fa-3db9-4f0d-8dce-623a15355ccd">

Removing the typo: "F" got rid of the error. I replicated it by putting it back, and I got the same error again with a delayed loading workspace dialog.

<img width="423" alt="image" src="https://github.com/microsoft/AL/assets/346940/5cdf228b-c6e6-4199-96e1-901e7c16a31a">

I hope that helps.

**Comment by sschuh** [comment]
Hi,

thanks for the suggestion.
I double checked my app.json's and can not find any typo or special characters.

If needed for analysis i will send my app.json's in here.

**Comment by kalberes** [comment]
Hi @sschuh 

I have tried this a few times on the latest 24.x and I have no repro. 
Do you have an active workspace? In other words do you have any AL file, or the app.json opened and has keyboard focus when you open your workspace?

**Comment by sschuh** [comment]
Hi,

nothing opened.

Version: v14.0.951286 (pre-release)

<img width="768" alt="image" src="https://github.com/microsoft/AL/assets/34685182/6319e3e6-f83f-4087-956a-925f658fc5bf">

The error comes before the loading of the workspace:
<img width="334" alt="image" src="https://github.com/microsoft/AL/assets/34685182/515acddf-336c-476e-8cfb-2b5001223ec4">

**Comment by fvet** [comment]
Same here when opening workspace files (no active editor / file opened)

![image](https://github.com/microsoft/AL/assets/25268332/96bba8b1-9d7b-449b-bbd1-4676a1efe0f7)

Name: AL Language extension for Microsoft Dynamics 365 Business Central
Id: ms-dynamics-smb.al
Description: Development tools for Dynamics 365 Business Central
Version: **14.0.951286**
Publisher: Microsoft
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=ms-dynamics-smb.al

**Comment by JesperSchulz** [comment]
The fix for this issue has been checked in to the master branch. It will be available in the bcinsider.azurecr.io/bcsandbox-master Docker image starting from platform build number 25.0.17048.0 and VS Code Extension Version 14.0.954663. 
 
 If you don’t have access to these images you need to become part of the Ready2Go program: aka.ms/readytogo 
 
 For more details on code branches and docker images please read: 
 https://blogs.msdn.microsoft.com/nav/2018/05/03/al-developer-previews-multiple-releases-and-github/ 
 https://freddysblog.com/2020/06/25/working-with-artifacts/

**Comment by fvet** [comment]
@JesperSchulz 

Any chance on backporting this to the current vsix? I'm running version 13.0.964488 and still encounter this issue, resulting in not being able to 'package' my apps in a multi project workspace.

`[Error - 8:53:38 AM] No active project was found. Make sure to select a project in the VS Code Explorer.`

**Comment by NKarolak** [comment]
I am affected only since updating to AL 13.0.
I could solve the issue by removing 
`"al.codeAnalyzers": [],`
from my User (!) settings file, restarting VS Code, and loading the first AL object.

If the BusinessCentral.LinterCop extension is installed, temporarily remove it from the `al.codeAnalyzers` setting (see also https://github.com/StefanMaron/BusinessCentral.LinterCop/issues/568).

**Comment by JesperSchulz** [comment]
> @JesperSchulz
> 
> Any chance on backporting this to the current 13.x vsix? I'm running the latest released version (13.0.964488) and now encounter this issue, resulting in not being able to 'package' my apps in a multi project workspace.
> 
> `[Error - 8:53:38 AM] No active project was found. Make sure to select a project in the VS Code Explorer.`

@qutreson, would that be doable?

**Comment by qutreson** [comment]
This fix has been backported to version 13 and will be part of the next release of the AL extension for VS code.
